src/Google/Ads/GoogleAds/Lib/OAuth2TokenBuilder.php

```diff
@@ -155,24 +169,64 @@ public function withImpersonatedEmail(?string $impersonatedEmail): self
      *
      * @return FetchAuthTokenInterface the created OAuth2 object that can fetch auth tokens
      */
-    public function build()
-    {
-        $this->defaultOptionals();
-        $this->validate();
 
-        if ($this->jsonKeyFilePath !== null) {
+    public function build(): FetchAuthTokenInterface
+    {
+        // The default scope for Google Ads API.
+        $adsScope = ['https://www.googleapis.com/auth/adwords'];
```

Comment:
Is this configurable? The scopes are configurable in some other libraries, so maybe this should come from a config variable?

Reply:
I added a comment above actually but the code was outdated:

```
I'm not 100% sure about the change here, but we do have object properties $this->scopes too.
Shouldn't we use that?
And if this variable is always used once here, why don't we just hard-code it below or declare a constant above in this class like other classes?
```

Reply:
Added more comments.